api/apps/restful_apis/chunk_api.py

```
@@ -506,13 +520,291 @@ async def get_chunk(tenant_id, dataset_id, document_id, chunk_id):
+        if isinstance(raw, str) and raw.strip():
+            return raw.strip()
+        return None
+
+    # Group: template_id → {entities, relations, kind}
+    grouped: dict[str, dict] = {}
+    for row in (rows or {}).values():
+        graph = {}
+        try:
+            graph = json.loads(row.get("content_with_weight") or "{}")
+        except Exception:
+            continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)